GitUI/UserControls/WaitSpinner.cs

```diff
@@ -130,8 +130,8 @@ protected override void OnPaint(PaintEventArgs e)
                 ref var angle = ref _angles[p];
                 e.Graphics.FillEllipse(
                     _brushes[i],
-                    (_centre.X - (_dotRadius / 2)) + (_circleRadius * angle.cos),
-                    (_centre.Y - (_dotRadius / 2)) + (_circleRadius * angle.sin),
+                    (_centre.X - (_dotRadius / 2f)) + (_circleRadius * angle.cos),
+                    (_centre.Y - (_dotRadius / 2f)) + (_circleRadius * angle.sin),
```

Comment:
Loss of precision

Reply:
See previous commend about subpixel rendering. At 96dpi this probably makes things look fuzzy.

Reply:
See previous comment about subpixel rendering. At 96dpi this probably makes things look fuzzy.